test/styles.css.js

```diff
@@ -1,6 +1,4 @@
-const styles = {
+module.exports = {
```

Comment:
Any specific reason for changing this to commonjs?

Reply:
The tests fails without the change:
```
● executes basic.js

    /Users/chris/Work/utils/babel-plugin-inline-classnames/test/styles.css.js:6
    export default styles;
    ^^^^^^

    SyntaxError: Unexpected token export

      at ScriptTransformer._transformAndBuildScript (node_modules/jest-runtime/build/script_transformer.js:316:17)
      at eval (eval at it (test/index.test.js:30:18), <anonymous>:5:38)
      at Object.it (test/index.test.js:30:18)
```

This ones also fails:
- executes custom-import-name.js
- executes object-expression.js
- executes string-literal-mixed.js